doc/source/whatsnew/v2.2.0.rst

```diff
@@ -365,7 +365,9 @@ I/O
 - Bug in :func:`read_excel`, with ``engine="xlrd"`` (``xls`` files) erroring when file contains NaNs/Infs (:issue:`54564`)
 - Bug in :func:`to_excel`, with ``OdsWriter`` (``ods`` files) writing boolean/string value (:issue:`54994`)
 - Bug in :meth:`pandas.read_excel` with an ODS file without cached formatted cell for float values (:issue:`55219`)
+- Bug in reading and writing ``datetime64`` dtypes with non-nanosecond resolution failing to round-trip correctly (:issue:`55622`)
```

Comment:
Should this mention `read_hdf`/`to_hdf`?

Reply:
yep, will update

Reply:
updated+green